### PR TITLE
Use config/images for image endpoint.

### PR DESCRIPTION
**Pull request by dman926**

### CODE REVIEW

File: api/src/routes/upload-image.ts [hunk 1]
```diff
@@ -99,35 +117,36 @@ export const uploadImageHandler: RouteHandlerMethod = async (req, rep) => {
       thumbnailOptions.map(genThumbnail(getFilename, sharpImage)),
     );
 
-    const savePromises = thumbnails
+    const validPromises = thumbnails
```
Comment (ZaxAtax) [inline api/src/routes/upload-image.ts:120]: This change is valid

**Review by ZaxAtax — APPROVED** [review]
😎

File: api/src/routes/upload-image.ts [hunk 2]
```diff
@@ -99,35 +117,36 @@ export const uploadImageHandler: RouteHandlerMethod = async (req, rep) => {
...
-            Promise.resolve(name),
-            supabase.storage.from('images').upload(`${loc}/${name}`, buffer, {
-              contentType,
-            }),
-          ]);
-        },
-      );
+      .map(({ value }) => value);
+
+    if (INCLUDE_ORIGINAL) {
```
Comment (john-rice) [inline api/src/routes/upload-image.ts:127]: Could also be broken up into a function and a call:


```
if (INCLUDE_ORIGINAL) {
  const originalFilename = getOriginalFilename();
  const formattedSharpImage: FormattedSharpImage = {
    name: originalFilename,
    buffer: imageBuffer,
    contentType: mFile.mimetype,
  };

  validPromises.push(formattedSharpImage);
}

const uploadImage = async ({ name, buffer, contentType }) => {
  const uploadPromise = supabase.storage.from('images').upload(`${loc}/${name}`, buffer, { contentType });
  return [name, await uploadPromise];
};

const savePromises = validPromises.map(uploadImage);

```

**Review by john-rice — APPROVED** [review]
Looks good. The one suggestion may be good to include if you agree it adds readability and testability.

**Pull request merged**